docker/serving_alpa.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# base docker image
+FROM nvidia/cuda:11.3.0-cudnn8-devel-ubuntu20.04
+
+# init workdir
+RUN mkdir -p /build
+WORKDIR /build
+
+# install common tool & conda
+RUN apt update && \
+    apt install wget -y && \
+    apt install git -y && \
+    apt install vim -y && \
+    wget --quiet https://repo.anaconda.com/archive/Anaconda3-2022.05-Linux-x86_64.sh -O ~/anaconda.sh && \
+    /bin/bash ~/anaconda.sh -b -p /opt/conda && \
+    rm ~/anaconda.sh && \
+    mkdir -p /opt/conda/envs/alpa && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc
+
+# install conda alpa env
+RUN . /opt/conda/etc/profile.d/conda.sh && \
+    conda create --name alpa python=3.8 -y && \
+    conda activate alpa && \
+    apt install coinor-cbc -y && \
+    pip3 install --upgrade pip && \
+    pip3 install cupy-cuda113 -i https://pypi.douban.com/simple && \
+    pip3 install alpa  -i https://pypi.douban.com/simple && \
+    pip3 install --trusted-host 169.229.48.123 -i http://169.229.48.123:8080/simple/ jaxlib==0.3.5+cuda113.cudnn820
```

Comment:
this is deprecated -- we now have a new pypi index

Reply:
Thanks for the reminder, I have updated the Dockerfile.